spec/integration/devtools/rake/flay/verify_spec.rb

```diff
@@ -49,7 +51,11 @@ def bar; end
 
   context 'when theshold is too low' do
     let(:instance) do
-      described_class.new(threshold: 0, total_score: 0, lib_dirs: directories, excludes: [])
+      described_class
+        .new(threshold: 0,
```

Comment:
This does not fit on how I'd write it these days. Lets go for:


```
described_class.new(
  threshold: 0,
  ...
)
```

If rubocop after the update wants what you did, please change the setting to make it comply.

Reply:
ok, will fix it today